meadow-client/src/Blockly.purs

```diff
@@ -0,0 +1,203 @@
+module Blockly where
+
+import Prelude
+
+import Blockly.Types (Block, Blockly, BlocklyState, Workspace)
+import Data.Function.Uncurried (Fn4, runFn4)
+import Data.Maybe (Maybe(..))
+import Data.Newtype (class Newtype, unwrap)
+import Data.Symbol (SProxy(..))
+import Data.Traversable (class Foldable, traverse_)
+import Effect (Effect)
+import Effect.Uncurried (EffectFn1, EffectFn2, EffectFn3, runEffectFn1, runEffectFn2, runEffectFn3)
+import Foreign (Foreign)
+import Halogen.HTML (AttrName(..), ElemName(..), Node)
+import Halogen.HTML.Elements (element)
+import Halogen.HTML.Properties (IProp, attr)
+import Record as Record
+import Simple.JSON (class WriteForeign)
+import Simple.JSON as JSON
+import Text.PrettyPrint.Leijen (align)
+
+foreign import createBlocklyInstance_ :: Effect Blockly
+
+foreign import createWorkspace_ :: EffectFn3 Blockly String String Workspace
+
+foreign import resizeBlockly_ :: EffectFn1 BlocklyState Unit
+
+foreign import addBlockType_ :: EffectFn3 BlocklyState String Foreign Unit
+
+foreign import initializeWorkspace_ :: EffectFn2 Blockly Workspace Unit
+
+foreign import render_ :: EffectFn1 Workspace Unit
+
+foreign import getBlockById_ :: forall a. Fn4 (a -> Maybe a) (Maybe a) Workspace String (Maybe Block)
+
+newtype Div
```

Comment:
This should really be called `ElementId`, right?

Reply:
renamed